### ISSUE TITLE
1.0.9 image history issues

### ISSUE THREAD

**Issue by kx1999** [savedra1/clipse#122]
[1.0.9 demonstration](https://streamable.com/ghxrxy)

Been having issues specifically with history of images in clipse since 1.0.3.
Using the AUR packages along with wl-clipboard on Hyprland.
1.0.2 works okay besides the issue from #101

**Comment by savedra1** [comment]
Hi @kx1999 👋

Thanks for reaching out about this!

I think I see what the issue is here. Your local time zone is being read differently to most by Go's datetime library, causing the program to incorrectly identify the images you copy as duplicates so only the most recent is being kept. 

If you check your `clipse.log` file you should see some errors saying the filename did not match the expected format when checking for duplicates. 

Image files should be saved in the following format: `<file binary size int>-<time stamp int>.png` without spaces. 

This will need to be a bug fix for the next release. The only temporary fixes I can think of are:
- using an older version like you mentioned you are already
- renaming any image files from the temp dir to match the above format
- changing the time zone on your system

Please can you confirm the time zone you currently have set so I can investigate if others might be affected? 

Thanks!

**Comment by kx1999** [comment]
Yes there are many errors with the same file format as this in `clipse.log`: 
```2024/08/24 10:54:55 logger.go:19: ERROR: could not get img identifier due to irregular filename | '260970-047767189 -0400 EDT m=.png'```

The currently set timezone is America/New_York 
```               Local time: Sun 2024-08-25 20:01:17 EDT
           Universal time: Mon 2024-08-26 00:01:17 UTC
                 RTC time: Mon 2024-08-26 00:01:18
                Time zone: America/New_York (EDT, -0400)
System clock synchronized: yes
              NTP service: active
          RTC in local TZ: no
```

**Comment by savedra1** [comment]
Thanks @kx1999 

After a little more investigation, this is caused by the daylight saving setting (EDT) that your system has been automatically set to rather than the time zone itself. If your time zone was set back to regular mode (EST) it would fix the issue, but that would mean you'd be an hour off unless manually overriding/using a different time zone, so not really a solution. 

I'll get a bug fix in for this as soon as I get the chance!

**Comment by savedra1** [comment]
Hi @kx1999 apologies for the delay but a fix has now been pushed out in v1.1.0 :smile:
